keyboards/planck/keymaps/snowkuma/keymap.c

```diff
@@ -0,0 +1,380 @@
+/* Copyright 2015-2017 Jack Humbert
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*                                   _
+ *  ___   _ __     ___   __      __ | | __  _   _   _ __ ___     __ _
+ * / __| | '_ \   / _ \  \ \ /\ / / | |/ / | | | | | '_ ` _ \   / _` |
+ * \__ \ | | | | | (_) |  \ V  V /  |   <  | |_| | | | | | | | | (_| |
+ * |___/ |_| |_|  \___/    \_/\_/   |_|\_\  \__,_| |_| |_| |_|  \__,_|
+ *
+ * https://github.com/snowkuma
+ *
+ * version 0.1
+*/
+
+#include QMK_KEYBOARD_H
+#include "muse.h"
+#include "custom_keycodes.h"
+#include "my_strings.h"
+
+extern keymap_config_t keymap_config;
+
+enum planck_layers {
+  _COLEMAK = 0,
+  _SYMBOL,
+  _SFT_NAV,
+  _REGEX,
+  _NUMBER,
+  _ARRANGE,
+  _FUNCTION,
+  _MOUSE,
+  _THUMB
+};
+
+enum planck_keycodes {
+  COLEMAK = SAFE_RANGE,
```

Comment:
As you aren't using the `COLEMAK` keycode, you could actually safely delete this definition, and its logic (Lines 301-306).

```suggestion
```

Reply:
That makes sense.  It is unused as I'm not switching between alpha layouts at all.  Many thanks.